src/pybind/mgr/dashboard/frontend/src/app/core/auth/login/login.component.scss

```diff
@@ -1,14 +1,17 @@
+@use 'sass:map';
 @use './src/styles/defaults/bootstrap-defaults' as bd;
 
+$dark-secondary: darken(bd.$secondary, 4%);
+
 ::ng-deep cd-login {
   .form-group {
-    background-color: bd.$color-password-toggle-bg;
-    border-left: 4px solid bd.$color-solid-white;
+    background-color: $dark-secondary;
+    border-left: 4px solid bd.$white;
     height: auto;
     margin-bottom: 2rem;
 
     &:focus-within {
-      border-left: 4px solid bd.$color-accent;
+      border-left: 4px solid map.get(bd.$theme-colors, 'accent');
```

Comment:
Actually it would be cleaner, but with the current implementation it could cause branding issues. `accent` is needed in the map to automatically genereate the different classes with the accent color like `btn-accent` or  `text-accent`.If any vendor wants to change the accent color, the map needs to be updated:
```
$theme-colors: (
  'accent': // Custom color
);
```
Currently, the vendor.scss must be imported after the `_bootstrap-defaults.scss` to overwrite variables, because the are not flagged as default. Imagine having a $accent variable, the vendor what have to change the value of the variable as well as the value of the map. At least I think that would be necessary, because at the time the vendor overwrites the variable the map would have been defined already.